phoenix-core/src/test/java/org/apache/phoenix/query/BaseTest.java

```diff
@@ -890,14 +901,17 @@ private static void deletePriorSchemas(long ts, String url) throws Exception {
         }
         try (Connection conn = DriverManager.getConnection(url, props)) {
             DatabaseMetaData dbmd = conn.getMetaData();
-            ResultSet rs = dbmd.getSchemas();
+            ResultSet rs = dbmd.getSchemas(null, "%" + phoenixTestTableName.getTableName() + "%");
```

Comment:
Unrelated to your changes, this code is needlessly complex. (And it's my fault).

This should should simply drop all HBase namespaces and tables, and let Phoenix re-initialize from scratch for the next test.